File: core/src/main/java/me/devsnox/planetsystem/core/commands/modules/InfoCommand.java
package me.devsnox.planetsystem.core.commands.modules;

import me.devsnox.planetsystem.api.player.PlanetPlayer;
import me.devsnox.planetsystem.core.commands.PlanetCommandModule;

public class InfoCommand implements PlanetCommandModule {

    @Override
    public void execute(final PlanetPlayer player, final String[] args) {
        if (player.getPlanet().getInner().isInside(player.getLocation())) {
            player.getLogger().success("commands.info.success");
        } else {
            player.getLogger().warn("commands.info.failed.not_own_planet");
        }
    }
}

File: core/src/main/kotlin/me/devsnox/planetsystem/core/commands/modules/InfoCommand.kt
package me.devsnox.planetsystem.core.commands.modules

import me.devsnox.planetsystem.api.location.isInside
import me.devsnox.planetsystem.api.player.PlanetPlayer
import me.devsnox.planetsystem.core.commands.PlanetCommandModule
import me.devsnox.planetsystem.core.log.MessageKeys

class InfoCommand : PlanetCommandModule {

	override fun execute(player: PlanetPlayer, args: Array<String>) {
		if (player.planet.inner.isInside(player.location)) {
			player.logger.success(MessageKeys.COMMANDS_INFO_SUCCESS)
		} else {
			player.logger.warn(MessageKeys.COMMANDS_INFO_FAILED_NOT_OWN_PLANET)
		}
	}
}
